Source/WebCore/layout/formattingContexts/inline/InlineContentConstrainer.h

```diff
@@ -36,6 +36,11 @@
 namespace WebCore {
 namespace Layout {
 
+enum class PreviousLineBreakStatus {
+    EndedWithLineBreak,
+    DidNotEndWithLineBreak
```

Comment:
are you planning on expanding this enum? If not you could just use `enum class PreviousLineEndsWithLineBreak : bool { No, Yes };`